flake.nix

```
@@ -54,15 +55,42 @@
             yarn install
           '';
         };
-      };
+      } // (
+        let
+          toDemoShell = version: contrast-release:
+            lib.nameValuePair "demo-${version}" (pkgs.mkShell {
+              packages = [ contrast-release ];
+
+              shellHook = ''
+                cd "$(mktemp -d)"
+
+                if [[ -e ${contrast-release}/runtime.yml ]]
+                then
+                  cp ${contrast-release}/runtime.yml .
+                fi
+
+                if [[ -e ${contrast-release}/coordinator.yml ]]
+                then
+                  cp ${contrast-release}/coordinator.yml .
+                fi
+                
+                if [[ -d ${contrast-release}/deployment ]]
+                then
+                  cp -r ${contrast-release}/deployment .
+                fi
+              '';
```

```suggestion
              shellHook = ''
                cd "$(mktemp -d)"
                [[ -e ${contrast-release}/runtime.yml ]] && cp ${contrast-release}/runtime.yml .
                [[ -e ${contrast-release}/coordinator.yml ]] && cp ${contrast-release}/coordinator.yml .
                [[ -d ${contrast-release}/deployment ]] && cp -r ${contrast-release}/deployment .
              '';
```
let's save some space.